checker/tests/ainfer-testchecker/non-annotated/all-systems/java21/Issue6173.java

```diff
@@ -1,26 +1,27 @@
 // @below-java21-jdk-skip-test
 
-// None of the WPI formats support the new Java 21 languages features, so skip inference until they do.
+// None of the WPI formats support the new Java 21 languages features, so skip inference until they
```

Comment:
These files in `checker/tests/ainfer-testchecker/non-annotated/all-systems/` don't exist in eisop. `all-systems` should just be a symbolic link. Can you fix this?

Reply:
@wmdietl done, see new commits.